OData/src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -244,9 +244,31 @@ private Expression CreateAggregationExpression(ParameterExpression accum, Aggreg
                         ExpressionHelperMethods.QueryableCountGeneric.MakeGenericMethod(propertyLambda.Body.Type);
                     aggregationExpression = Expression.Call(null, countMethod, distinctExpression);
                 }
-                    break;
+                break;
                 default:
```

Comment:
Seems like all this code should be under "AggregationMethodType.Custom" and default should remain as-is.